src/App.css

```diff
@@ -157,5 +157,25 @@ p{
 }
 
 .bgcolor{
-  background-color: #495056
-}
+  background-color: #495056;
+}
+.not-found{
+  display: flex;
+  height: 880px;
+  background: url('./images/404.jpg') no-repeat;
+  background-size: cover;
+  padding: 9% 0 4% 0;
+}
+.not-found-container{
+  bottom: 40%;
+  margin: auto;
+  position: relative;
+}
+.btn-not-found{
+  background-color: #495056;
+  border-color: white;
+  margin-bottom: -287%!important;
+}
+.error-template{
+  margin-top: 10px!important;
+}
```

Comment:
Please add the newline at the end of the file

Reply:
Done, thanks for the reminder